docs/DevGuideFeatureImplementations/AddFeatureGuide.adoc

```diff
@@ -0,0 +1,44 @@
+=== Add feature
+==== Implementation
+
+The `add` feature uses the Model to check if the dish is already in the database.
+If the dish is already in the database, the meal will be added to the meal log only.
+If the dish the user wants to add is not in the database,
+then the dish will be added to both the meal log and the dish database so that the user
+can conveniently add it the next time.
+
+There are a few cases to take note:
+
+* If the dish is in the dish database:
+** The user inserts the calorie information that is the same as the one in the dish database
+*** The dish will be added to the meal log only.
```

Comment:
Do you want to consider standardising your punctuation (i.e. your full stops and colons)?